core/domain/skill_services_test.py

```diff
@@ -90,6 +99,27 @@ def test_get_skill_summary_by_id(self):
         self.assertEqual(skill_summary.description, 'Description')
         self.assertEqual(skill_summary.misconception_count, 1)
 
+    def test_get_skill_mastery(self):
```

Comment:
Do not merge the skill mastery stuff into this test suite -- I suggest writing a different test suite altogether so that you have a bit more separation. Otherwise this test suite will become too large and complicated.